.circleci/config.yml

```diff
@@ -18,11 +18,18 @@ defaults: &defaults
 
 jobs:
   install_dependencies:
-    - run: echo "NOOP"
+    <<: *defaults
+    steps:
+      - run: echo "SKIP"
   lint-checks:
-    - run: echo "NOOP"
+    <<: *defaults
+    steps:
+      - run: echo "SKIP"
+
   general-test:
-    - run: echo "NOOP"
+    <<: *defaults
+    steps:
+      - run: echo "SKIP"
```

Comment:
Why not get rid of the CircleCI configs completely?

Reply:
I tried removing it, but there's a CI webhook on the I don't have permissions to remove, and if I deleted the CI file, it fails because it can't find the config file. I just did this for now to make the CI quiet and was gonna ask whoever has owner permissions on the repo to disable it and then we can remove it. If you can do that it would be great!